catalog/transformer/catalog-transformer-pptx/src/test/java/ddf/catalog/transformer/input/pptx/PptxInputTransformerTest.java

```diff
@@ -114,10 +114,10 @@ public void testWithTitle()
                 ss.write(os);
 
                 try (ByteArrayInputStream inStr = new ByteArrayInputStream(os.toByteArray())) {
-                    TikaInputTransformer inputTransformer = new TikaInputTransformer(null,
+                    TikaInputTransformer realTransformer = new TikaInputTransformer(null,
```

Comment:
Checkstyle complained that `inputTransformer` was shadowing another variable, so I renamed it to `realTransformer` (The `inputTransformer` is a mock).